src/Microsoft.PowerShell.Commands.Management/commands/management/GetClipboardCommand.cs

```diff
@@ -211,4 +269,109 @@ private List<PSObject> GetClipboardContentAsFileList()
             return result;
         }
     }
+
+#if UNIX
+    internal class ClipboardHelper
+    {
+        private static bool? _clipboardSupported;
+
+        private static string StartProcess(
+            string tool,
+            string args,
+            string stdin = ""
+        )
+        {
+            ProcessStartInfo startInfo = new ProcessStartInfo();
+            startInfo.UseShellExecute = false;
+            startInfo.RedirectStandardInput = true;
+            startInfo.RedirectStandardOutput = true;
+            startInfo.RedirectStandardError = true;
+            startInfo.FileName = tool;
+            startInfo.Arguments = args;
+            string stdout = null;
+
+            using (Process process = new Process())
+            {
+                process.StartInfo = startInfo;
+                try
+                {
+                    process.Start();
+                }
+                catch (System.ComponentModel.Win32Exception)
+                {
+                    _clipboardSupported = false;
+                    throw new PlatformNotSupportedException();
+                }
+
+                if (stdin != "")
+                {
+                    process.StandardInput.Write(stdin);
+                    process.StandardInput.Close();
+                }
+                stdout = process.StandardOutput.ReadToEnd();
+                process.WaitForExit(250);
+
+                _clipboardSupported = process.ExitCode == 0;
+            }
+
+            return stdout;
+        }
+
+        public static string GetText()
+        {
+            if (_clipboardSupported == false)
+            {
+                throw new PlatformNotSupportedException();
+            }
+
+            string tool = "";
+            string args = "";
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                tool = "xclip";
+                args = "-selection clipboard -out";
```

Comment:
We could just do here `return StartProcess("xclip", "-selection clipboard -out");`.
And the same for MacOs below.